tst/Encamina.Enmarcha.AI.Tests/RecursiveCharacterTextSplitterTests.cs

```diff
@@ -82,6 +82,51 @@ public void SplitText_WithParameterTextSplitterOptions_NotAlter_DefaultTextSplit
         Assert.Equal("tortor vitae purus faucibus ornare suspendisse.", splitsWithDefaultOptions[9]);
     }
 
+    [Fact]
+    public void SplitText_WithEmptyString_ReturnsNoSplits()
+    {
+        // Arrange
+        var defaultTextSplitterOptions = new TextSplitterOptions
+        {
+            Separators = new List<string> { "\n\n", "\n", " ", string.Empty },
+            ChunkOverlap = 0,
+            ChunkSize = 100,
+        };
+        var optionsMonitor = new TestOptionsMonitor<TextSplitterOptions>(defaultTextSplitterOptions);
+        var recursiveCharacterTextSplitter = new RecursiveCharacterTextSplitter(optionsMonitor);
+        var lengthFunction = ILengthFunctions.LengthByCharacterCount;
+        var text = string.Empty;
+
+        // Act
+        var splits = recursiveCharacterTextSplitter.Split(text, lengthFunction).ToList();
+
+        // Assert
+        Assert.Empty(splits);
+    }
+
+    [Fact]
+    public void SplitText_WithSingleWord_ReturnsSingleSplit()
+    {
+        // Arrange
+        var defaultTextSplitterOptions = new TextSplitterOptions
+        {
+            Separators = new List<string> { "\n\n", "\n", " ", string.Empty },
+            ChunkOverlap = 0,
+            ChunkSize = 100,
+        };
+        var optionsMonitor = new TestOptionsMonitor<TextSplitterOptions>(defaultTextSplitterOptions);
+        var recursiveCharacterTextSplitter = new RecursiveCharacterTextSplitter(optionsMonitor);
+        var lengthFunction = ILengthFunctions.LengthByCharacterCount;
+        var text = "Word";
+
+        // Act
+        var splits = recursiveCharacterTextSplitter.Split(text, lengthFunction).ToList();
+
+        // Assert
+        Assert.Single(splits);
+        Assert.Equal("Word", splits[0]);
```

Comment:
```suggestion
        var split = Assert.Single(splits);
        Assert.Equal("Word", split);
```
Since `Assert.Single` returns the single element (if it doesn’t throw an exception), you can use it directly in the next assertion. Just a tip—it's not necessary to change it :)